build.sbt

```diff
@@ -44,7 +44,15 @@ lazy val metaserver = project
     ),
     resolvers += "dhpcs at bintray" at "https://dl.bintray.com/dhpcs/maven",
     testFrameworks := new TestFramework("utest.runner.Framework") :: Nil,
+    fork in Test := true,
+    buildInfoKeys := Seq[BuildInfoKey](
+      "testWorkspaceBaseDirectory" ->
```

Comment:
Handy, thanks.

Reply:
This makes the tests agnostic to sys.props(user.dir)